components/ai_chat/core/browser/ai_chat_database.cc

```diff
@@ -77,14 +78,17 @@ std::vector<mojom::ConversationPtr> AIChatDatabase::GetAllConversations() {
 std::vector<mojom::ConversationEntryPtr> AIChatDatabase::GetConversationEntries(
     int64_t conversation_id) {
   sql::Statement statement(GetDB().GetUniqueStatement(
-      "SELECT conversation_entry.*, entry_text.* "
-      "FROM conversation_entry"
-      " JOIN ("
-      " SELECT *"
-      " FROM conversation_entry_text"
-      ") AS entry_text"
-      " ON conversation_entry.id = entry_text.conversation_entry_id"
-      " WHERE conversation_id=?"
+      "SELECT conversation_entry.id, conversation_entry.date, "
+      "conversation_entry.character_type, conversation_entry.action_type, "
+      "conversation_entry.selected_text, text.id, text.date, text.text, "
+      "text.conversation_entry_id, "
+      "GROUP_CONCAT(DISTINCT search_queries.query) AS search_queries"
+      " FROM conversation_entry"
+      " JOIN conversation_entry_text as text"
+      " JOIN search_queries"
```

Comment:
We're not specifying a `WHERE` to specify how the `search_queries` table should be joined - how does that work?

Reply:
i have no idea but makes me think i need more strict tests when verifying `search_queries`